apps/checker/app/views/rules.scala.html

```diff
@@ -27,56 +27,13 @@ <h5>Matchers (@matchers.size)</h5>
                     </tr>
                 </thead>
                 <tbody>
-                @for(matcher <- matchers) {
+                @for(matcher <- matchers.sortBy(matcher => 0 - matcher.getRules().size)) {
```

Comment:
```suggestion
                @for(matcher <- matchers.sortBy(0 - _.getRules().size)) {
```

(arguably less readable, please feel free to ignore!)